docs/contributing/pull-requests/branching.md

```diff
@@ -1,4 +1,5 @@
 ---
+sidebar_position: 1
```

Comment:
Why are we setting sidebar positions on these? Do we dislike the alphabetical ordering?